### PR TITLE
Pricing page: update bundles copy

### PR DESCRIPTION
**Pull request by monsieur-z**
### Changes proposed in this Pull Request

This PR updates the copy of the Security and Complete plans, in the Bundles section.

### Implementation notes

Here's the list of updates:

#### Comparison table

- Real-time backups as you edit (both sides)
  -> VaultPress Backup: Real-time backups as you edit
- Real-time malware scanning and one-click fixes (both sides)
  -> Scan: Real-time malware scanning and one-click fixes
- Comment and form spam protection 
  -> Akismet Anti-spam: Comment and form spam protection (10k API calls/mo)
  -> Akismet Anti-spam: Comment and form spam protection (60k API calls/mo)
  If it doesn’t fit, remove the word ‘Anti-spam’
- Boost: Automatic CSS Generation
  -> Boost: Automatic CSS generation
- Add new row Under Growth (above CRM)
  - Social: Basic with 1,000 shares/mo
  - [Red x on left column, green check on right column]

#### Security modal
- [Underneath Easy-to-use paragraph]
  - Add “Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam”
- Real-time backups as you edit
  -> VaultPress Backup: Real-time backups as you edit
- Real-time malware scanning and one-click fixes
  -> Scan: Real-time malware scanning and one-click fixes
- Comment and form spam protection
  -> Akismet Anti-spam: Comment and form spam protection (10k API calls/mo)

#### Complete modal
- Real-time backups as you edit
  -> VaultPress Backup: Real-time backups as you edit
- Real-time malware scanning and one-click fixes
  -> Scan: Real-time malware scanning and one-click fixes
- Comment and form spam protection 
  -> Akismet Anti-spam: Comment and form spam protection (10k API calls/mo)
- VideoPress with 1TB of ad-free video hosting
  -> VideoPress: 1TB of ad-free video hosting
- [Add row] Boost: Automatic CSS generation
- Site Search up to 100k records
  -> Site Search: Up to 100k records
- [Add row] Social: Basic with 1,000 shares/mo
- CRM Entrepreneur
  -> CRM: Entrepreneur with 30 extensions

### Testing instructions

- Spin up the pricing page, using a live link below, or by running this branch locally
- Check that the aforementioned updates are reflected in the page

### CODE REVIEW

**Comment by jkguidaven** [comment]
I am not sure if it was intended not to include **"Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam"** in the featured card description but I assume the paragrah here is the shorter version?

Screen Shot 2022-11-15 at 8 10 04 PM

File: packages/calypso-products/src/plans-list.tsx [hunk 1]
```diff
@@ -1168,7 +1168,7 @@ const getPlanJetpackSecurityT1Details = (): IncompleteJetpackPlan => ( {
 		),
 	getLightboxDescription: () =>
 		translate(
-			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection.'
+			'Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection. Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.'
```
Comment (jkguidaven) [inline packages/calypso-products/src/plans-list.tsx]: I feel like the message here is redundant `including backups, malware scanning, and spam protection. Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.`

Should we just simplify it like below?

```
Easy-to-use, comprehensive WordPress site security including
VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.
```

**Comment by monsieur-z** [comment]
> I am not sure if it was intended not to include **"Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam"** in the featured card description but I assume the paragrah here is the shorter version?
> 
> Screen Shot 2022-11-15 at 8 10 04 PM

Yes, the added sentence is just in the lightbox.

File: packages/calypso-products/src/plans-list.tsx (hunk 1 above)
Comment (monsieur-z) [reply packages/calypso-products/src/plans-list.tsx]: Good call. I've added a line break, I think it helps.

**Review by jkguidaven — APPROVED** [review]
Working as expected 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7626332

Hi @monsieur-z, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `VaultPress Backup: Real-time backups as you edit`
- `Easy-to-use, comprehensive WordPress site security including backups, malware scanning, and spam protection.{{br/}}Includes VaultPress Backup, Jetpack Scan, and Akismet Anti-spam.`
- `Boost: Automatic CSS generation`
- `Akismet: Comment and form spam protection (10k API calls/mo)`

Thank you in advance!

**Comment by niranjan-uma-shankar** [comment]
If these pages can be loaded in locales other than english, then translations would break across all the strings modified in this PR. Non-en users would see english language language text in the middle of other translated content.

To provide a good experience for non-EN users, I would recommend that you revert this PR, restart this PR with `String freeze` label, await translations to complete, and then deploy the change. Alternatively, deploy the string change only for `en` locales or if a translation already exists (using [hasTranslation()](https://github.com/Automattic/wp-calypso/blob/a93b7275a2cc78c48ee6f5ab0b13be07788f235e/packages/i18n-calypso/README.md#hastranslation-method)).

**Comment by CodeyGuyDylan** [comment]
@monsieur-z 
@elliottprogrammer was asking about these changes and we noticed that the new copy for the features of Jetpack Complete say `Social: Basic with 1000 shares/mo`, but Based on the product card on jetpack.com/social, it seems as though this should say unlimited?
image

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7626332).

**Pull request merged**